src/ko/ui/defaultViewManager.html

```diff
@@ -33,7 +34,7 @@
 </div>
 
 <!-- ko with: widgetEditor() -->
-<div class="toolbox toolbox-position-left-offset" data-bind="resizable: resize, surface: $data, dialog: {}">
+<div class="toolbox toolbox-position-left-offset" data-bind="visible: !$component.previewMode(), resizable: resize, surface: $data, dialog: {}">
```

Comment:
There is API in DefaultViewManager to hide editors, so we don't really need this in the View:
`data-bind="visible: !$component.previewMode()`. Just look what changes `widgetEditor()` observable. Also, take a look at the methods like: 
`hideToolboxes()`, `designTime()`, `closeWorkshop()`, `clearContextualEditors()`